models/orderDao.js

```diff
@@ -0,0 +1,71 @@
+const { database } = require('./dataSource');
+
+const orderInfo = async ( userId ) => { 
+  const orderInfo = await database.query( 
+    `
+    SELECT 
+      p.id AS productId,
+      p.name AS productName,
+      p.thumbnail_image_url AS ProductThumbnailImageUrl,
+      p.price AS productPrice,
+      s.size AS Size,
+      oi.quantity AS quantity,
+      u.point AS point
+
+      FROM orders o
+      JOIN order_items oi ON oi.order_id = o.id
+      JOIN stock s ON oi.stock_id = s.id
+      JOIN products p ON s.product_id = p.id
+      JOIN users u ON o.user_id = u.id
+      WHERE u.id = ?
+     `,[userId]
+  );
+   return orderInfo;
+};
```

Comment:
함수는 `Action`의 의미를 갖고 있기 때문에 함수 이름은 명사가 아니라 동사로 만들어주세요.

```suggestion
const getOrder = async ( userId ) => { 
  const orderInfo = await database.query( 
    `
    SELECT 
      p.id AS productId,
      p.name AS productName,
      p.thumbnail_image_url AS ProductThumbnailImageUrl,
      p.price AS productPrice,
      s.size AS Size,
      oi.quantity AS quantity,
      u.point AS point

      FROM orders o
      JOIN order_items oi ON oi.order_id = o.id
      JOIN stock s ON oi.stock_id = s.id
      JOIN products p ON s.product_id = p.id
      JOIN users u ON o.user_id = u.id
      WHERE u.id = ?
     `,[userId]
  );
   return orderInfo;
};
```